### ISSUE TITLE
Warning about default exports for rollup.config.js

### ISSUE THREAD

**Issue by arcfide** [sveltejs/sapper-template#238]
I am seeing the following warning for rollup.config.js in my output when running `sapper run dev`.

```
Entry module "rollup.config.js" is implicitly using "default" export mode, which means for CommonJS 
output that its default export is assigned to "module.exports". For many tools, such CommonJS output 
will not be interchangeable with the original ES module. If this is intended, explicitly set "output.exports" 
to either "auto" or "default", otherwise you might want to consider changing the signature of "rollup.config.js" 
to use named exports only.
```

What is appropriate in this case? Should I just set `output.exports` as indicated or would it be better to switch over rollup.config.js to use named exports? This is a part of the sapper infrastructure that I don't fully have confidence in tweaking myself at this level, yet.

**Comment by jbg** [comment]
https://github.com/sveltejs/sapper/pull/1331

**Comment by Conduitry** [comment]
This should be fixed as of Sapper 0.28.
